README.md

```diff
@@ -89,15 +89,15 @@ Name                | Types           | Description
 
 #### Examples
 
-To install PostgreSQL server, set you own postgres password using non-default service port.
+To install PostgreSQL server, set your own postgres password using non-default service port.
 
-```
+```ruby
 postgresql_server_install 'My PostgreSQL Server install' do
   action :install
 end
 
-postgresql_server_install 'Setup my PostgreSQL 9.5 server' do
-  password 'MyP4ssw0d'
+postgresql_server_install 'Setup my PostgreSQL 9.6 server' do
```

Comment:
9.6 is currently the default, so that's the correct name I think.

Reply:
do we want to update all the examples to 9.6 then?

Reply:
Well, the _default_ is 9.6 and this example doesn't define a version, that's why I wrote 9.6 here. TBH it might be best to remove the version here because it's just a string and chances that this won't get maintained when the default changes are probably very high.

Where 9.5 is used it's to show that different versions can be installed as well. Personally I'd change those to 10.6 or 11.1
https://www.postgresql.org/support/versioning/

On that note, the README says we support 9.3, while 9.3 is now officially unsupported. I'm not sure what to do in this case. Either the README should not mention specific version numbers (since I saw no tests actually testing 9.3 or even 10 or 11) or we need to release an update?

Reply:
Yeah we totally don't support 9.3 it should just say we support https://www.postgresql.org/support/versioning/

which is what I thought I put! :(

Reply:
Ah it does. But I then say, we support this minor version. 

Can you drop that bit off to please?